primitive/src/main/java/io/atomix/primitive/partition/impl/DefaultPartitionService.java

```diff
@@ -49,13 +49,14 @@ public PartitionGroup getPartitionGroup(String name) {
 
   @Override
   @SuppressWarnings("unchecked")
-  public Collection<PartitionGroup> getPartitionGroups() {
+  public Collection<PartitionGroup<?>> getPartitionGroups() {
```

Comment:
Why wildcard?

Reply:
Without the wildcard I have to do a lot of casting when using the streams API to iterate through groups and partitions